src/components/InputTag/InputTag.tsx

```diff
@@ -1,16 +1,39 @@
-import { DetailedHTMLProps, InputHTMLAttributes } from "react"
-import style from "./InputTag.module.scss"
+import { DetailedHTMLProps, InputHTMLAttributes } from "react";
+import style from "./InputTag.module.scss";
+import { isNumberValid } from "./validation";
 
 type Props = {
-    label: string,
-    isOn: boolean,
-} & DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>
+    id: string;
+    isOn: boolean;
```

Comment:
isOn is redundant because you have the disable property, but disable might be undefined (DetailedHTMLProps<InputHTMLAttribute.....> says it can be). So to fix this, add "disabled" to the list of omitted properties and put it in the Props type yourself.